dist/index.html

```diff
@@ -2,7 +2,7 @@
 <html class="no-js" lang="">
     <head>
         <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta http-equiv="x-ua-compatible" content="IE=edge">
```

Comment:
Maybe keep the PR focussed? It's something with a separated issue, where first testing needs to happen.